spec/lib/action_kit_rest/response/collection_spec.rb

```diff
@@ -7,8 +7,8 @@
 
   describe 'initialization' do
     it 'should set meta and objects' do
-      subject.meta.should == meta
-      subject.objects.should == objects
+      expect(subject.meta).to eq(meta)
```

Comment:
There's something we can configure in `spec_helper` to only support the `expect` syntax, right?

Reply:
Yes, but the previous syntax was not actually testing anything either I don't think. This switches us to expects, while also actually testing the behavior, which is the more relevant change.